www-apps/icingaweb2-module-director/icingaweb2-module-director-1.10.2.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+DESCRIPTION="Icinga Web 2 plugin for configuration"
+HOMEPAGE="https://github.com/Icinga/icingaweb2-module-director/"
+if [[ "${PV}" == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Icinga/${PN}.git"
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://codeload.github.com/Icinga/${PN}/tar.gz/v${PV} -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-${PV}"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+
+# Dependencies according to https://github.com/Icinga/icingaweb2-module-director/blob/master/doc/02-Installation.md
+IUSE="php_targets_php8-1"
+PHP_DEPEND="
+	php_targets_php8-1? ( dev-lang/php:8.1[curl,iconv,pcntl,posix,sockets] )
+"
+RDEPEND="
+	${PHP_DEPEND}
+	>=www-apps/icingaweb2-module-incubator-0.18.0
+	>=dev-libs/icinga-php-library-0.5.0
+	>=dev-libs/icinga-php-thirdparty-0.9.0
+	>=www-apps/icingaweb2-2.9.0
+	"
+DEPEND="
+	acct-group/icingaweb2
+	acct-user/icingadirector
```

Comment:
users only required at build time, but not runtime? :raised_eyebrow:

Reply:
I have no excuse for that one, sorry... fixed it